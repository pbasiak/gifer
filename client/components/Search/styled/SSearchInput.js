import styled from 'styled-components';

export const SSearchInput = styled.input`
    padding: 13px 20px;
    border: 2px solid #000;
    margin-right: 10px;
`;
