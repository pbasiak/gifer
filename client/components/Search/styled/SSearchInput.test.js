import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { SSearchInput } from './SSearchInput';
import 'jest-styled-components';

describe('<SSearchInput/>', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(<SSearchInput />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
})
