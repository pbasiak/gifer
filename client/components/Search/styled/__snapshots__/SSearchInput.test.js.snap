// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SSearchInput/> should match snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
    padding: 13px 20px;
    border: 2px solid #000;
    margin-right: 10px;
",
        ],
      },
      "displayName": "styled.input",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "input",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
